### ISSUE TITLE
All tooltips/titletips for UI elements in the Customize Dashboard read `Customize Dashboard`

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#20860]
## Description 

All tooltips/titletips for UI elements in the Customize Dashboard read `Customize Dashboard`

## Steps to Reproduce

   1. new profile
   2. launch Brave
   3. open a new-tab page
   4. click on `Customize` on the bottom right
   5. hover on the various UI elements: toggle switches, images, close widget, etc.

## Actual result:

They are literally all `Customize Dashboard`

## Expected result:

For accessibility purposes, they should be specific to the UI element/its function

## Reproduces how often: 

100%

## Brave version (brave://version info)

Brave | 1.36.80 Chromium: 98.0.4758.74 (Official Build) beta (x86_64)
-- | --
Revision | d0fe1ec4df090cd3eb02b591228505e12ea476e9-refs/branch-heads/4758@{#935}
OS | macOS Version 11.6.1 (Build 20G224)

(on `nightly` too)

## Version/Channel Information:

- Can you reproduce this issue with the current release? `yes`
- Can you reproduce this issue with the beta channel? `yes`
- Can you reproduce this issue with the nightly channel? `yes`

cc @petemill @simonhong @rebron
